packages/lisk-passphrase/package.json

```diff
@@ -61,24 +61,26 @@
 		"bip39": "2.5.0"
 	},
 	"devDependencies": {
-		"babel-cli": "6.26.0",
-		"babel-plugin-istanbul": "5.0.1",
-		"babel-plugin-module-resolver": "3.1.1",
-		"babel-plugin-transform-runtime": "6.23.0",
-		"babel-preset-env": "1.7.0",
-		"babel-register": "6.26.0",
+		"@types/bip39": "2.4.0",
+		"@types/chai": "4.1.5",
+		"@types/expect": "1.20.3",
+		"@types/jquery": "3.3.9",
```

Comment:
Is jquery being used in this lib?

Reply:
I tested after removing this dependency and all the tests are passing. Should I remove it or keep it just in case?

Reply:
then it's good to delete

Reply:
Done

Reply:
I tested it again, I see that it is throwing an error in the browser test, so I think we need it.